drivers/adc/Kconfig.max32

```diff
@@ -0,0 +1,11 @@
+# ADI SoC ADC configuration options
+
+# Copyright (c) 2023 Analog Devices, Inc.
```

Comment:
2023-2024 would be better.
Please apply it for all cases in this PR.

Reply:
Updated, thanks.